license-classifications.yml

```diff
@@ -149,9 +149,14 @@ categories:
       This property indicates that the license requires that the source code of the OSS is either
       distributed, or offered to distribute, in connection with the software delivery.
 
-  - name: "property:artistic-license"
+  - name: "property:modification-related-obligations"
     description: >-
-      This property is included only in the Artistic licenses.
+      This property indicates that the license contains obligations that are triggered by modifying
+      the licensed content. The scope of obligations could vary, but typically it is about somehow
+      making sure that the recipients of the modified content don't attribute the modifications to
+      the wrong author and/or that the unmodified content is available to the recipients. The
+      obligations go beoynd of mere marking of the modifications. Artistic-type licenses have this
+      property, but also some others.
```

Comment:
Is the category of Artistic-type licenses large and so property defining, that it should be separately mentioned here, in addition to using it in their classification?

Reply:
We currently have 4 versions of artistic licenses and MITNFA. So a total of 5 instances of this new/changed property. Considering we had a property artistic, this is also reasonably background. Further, artistic lienses are well-known to have this type of properties, whereas MITNFA was totally new to me, at least. So worth spending one line in classification for this.